tests/from.bats

```diff
@@ -439,3 +445,52 @@ load helpers
 #  run_buildah run $cid arch
 #  expect_output "s390x"
 }
+
+@test "from --authfile test" {
+  _prefetch busybox
+  run_buildah login --tls-verify=false --authfile ${TESTDIR}/test.auth --username testuser --password testpassword localhost:5000
+  run_buildah push --signature-policy ${TESTSDIR}/policy.json --tls-verify=false --authfile ${TESTDIR}/test.auth busybox docker://localhost:5000/buildah/busybox:latest
+  target=busybox-image
+  run_buildah from -q --signature-policy ${TESTSDIR}/policy.json --tls-verify=false --authfile ${TESTDIR}/test.auth docker://localhost:5000/buildah/busybox:latest
+  run_buildah rm $output
+  run_buildah rmi localhost:5000/buildah/busybox:latest
+}
+
+@test "from --cap-add/--cap-drop test" {
+  _prefetch alpine
+  # Try with default caps.
+  run_buildah from --quiet --pull=false --signature-policy ${TESTSDIR}/policy.json alpine
+  cid=$output
+  run_buildah run $cid grep ^CapEff /proc/self/status
```

Comment:
You could write this as:
```bash
  CAP_DAC_OVERRIDE=2   (unlikely to change)

  run_buildah run $cid awk '/^CapEff:/ { print $2}' /proc/self/status
  defaultcaps="$output"
  ...
  if (( 0x$defaultcaps & $CAP_DAC_OVERRIDE)); then
    run_buildah ... --cap-drop ...
    if (( 0x$droppedcaps & $CAP_DAC_OVERRIDE)); then
      die "--cap-drop did not drop DAC_OVERRIDE: $droppedcaps"
    fi
  else
    run_buildah ... --cap-add
   ...
```
Probably not likely ever to trigger, but if it does, this could give a helpful error message that might help a developer understand where to start looking in the code.